src/main/java/seedu/duke/util/StringConstants.java

```diff
@@ -196,6 +197,7 @@
             + "View all available config settings with \"option\".";
     public static final String ERROR_MODULE_LIST_EMPTY = "Sorry, you have 0 MCs counted towards your GPA ._.\n"
             + "Please add some modules or grades!";
+    public static final String ERROR_WRONG_DURATION_FORMAT = "Sorry, the estimated time is in wrong format Lol";
```

Comment:
this Lo1 looks like lol instead so im not sure if u want to keep it

Reply:
okk